.travis.yml

```diff
@@ -7,7 +7,7 @@ language: python
 cache: pip
 
 python:
-#  - "3.7"
+  - "3.7"
```

Comment:
Currently "nightly" already corresponds to 3.7.2 so I don't think this is usefull for the moment.

Reply:
+1